examples/get_started/03_figure_element.py

```diff
@@ -0,0 +1,92 @@
+"""
+3. Figure elements
+===============
+The figure shows the naming of figure elements in PyGMT.
+
+- :meth:`pygmt.Figure`: having a number of plotting methods. Every plot or map
+  must start with the creation of a :meth:`pygmt.Figure` instance
+- ``frame``: setting map boundary (**WSNE** or **wsne**), annonate and grid
+  (**afg**), axis label (**+l**) and title (**+t**) in
+  :meth:`pygmt.Figure.basemap`. The detail examples can be found at
+  `frame and axes attributes<https://www.pygmt.org/latest/tutorials/basics/
+  frames.html>`_.
```

Comment:
Beside `pygmt.Figure.basemap` also other methods have the ``frame`` parameter.
```suggestion
- `
- ``frame``: setting plot boundaries (**WSNE**, **wsne**, or **lbtr**),
   adding annotations, ticks, gridlines (**afg**), axis labels (**+l**),
   and title (**+t**), e.g., in :meth:`pygmt.Figure.basemap`. Detailed
   examples can be found at
   `frame and axes attributes<https://www.pygmt.org/latest/tutorials/basics/frames.html>`_.
```